### ISSUE TITLE
Fulfillment Inbound 2024-03-20:  List Placement Options returns different results for same InboundPlanId 

### ISSUE THREAD

**Issue by brandondev1** [amzn/selling-partner-api-models#3983]
I execute ListPlacementOptions for a particular InboundPlanId twice in my workflow (I only call GeneratePlacementOptions once).  The first time I call it, I get three placement options (strangely they all are only one shipment with no fees).  

`{
  "pagination": {},
  "placementOptions": [
    {
      "fees": [
        {
          "description": "Placement service fee represents service to inbound with minimal shipment splits and destinations of skus",
          "type": "FEE",
          "value": {
            "amount": 0,
            "code": "USD"
          },
          "target": "Placement Services"
        }
      ],
      "shipmentIds": [
        "sh1e303073-05e4-480e-8575-596e90508167"
      ],
      "discounts": [],
      "expiration": "2024-06-23T18:27Z",
      "placementOptionId": "pl01b71b43-397c-438d-a6be-6afa4a6895a0",
      "status": "OFFERED"
    },
    {
      "fees": [
        {
          "description": "Placement service fee represents service to inbound with minimal shipment splits and destinations of skus",
          "type": "FEE",
          "value": {
            "amount": 0,
            "code": "USD"
          },
          "target": "Placement Services"
        }
      ],
      "shipmentIds": [
        "shb7da8f62-8c8e-41d2-9dc3-9e01ecd51e18"
      ],
      "discounts": [],
      "expiration": "2024-06-23T18:27Z",
      "placementOptionId": "pl7e709784-a55e-4000-b21d-effec5f4c12a",
      "status": "OFFERED"
    },
    {
      "fees": [
        {
          "description": "Placement service fee represents service to inbound with minimal shipment splits and destinations of skus",
          "type": "FEE",
          "value": {
            "amount": 0,
            "code": "USD"
          },
          "target": "Placement Services"
        }
      ],
      "shipmentIds": [
        "shced4cc5c-ed28-4fa8-8b5d-528043ebf776"
      ],
      "discounts": [],
      "expiration": "2024-06-23T18:27Z",
      "placementOptionId": "ple8c69e03-6ace-46bd-a261-663ddab40d5e",
      "status": "OFFERED"
    }
  ]
}`

The second time I get only one of the three placement options (this time it has a fee):

`{
  "pagination": {},
  "placementOptions": [
    {
      "fees": [
        {
          "description": "Placement service fee represents service to inbound with minimal shipment splits and destinations of skus",
          "type": "FEE",
          "value": {
            "amount": 153,
            "code": "USD"
          },
          "target": "Placement Services"
        }
      ],
      "shipmentIds": [
        "shced4cc5c-ed28-4fa8-8b5d-528043ebf776"
      ],
      "discounts": [],
      "expiration": "2024-06-23T18:27Z",
      "placementOptionId": "ple8c69e03-6ace-46bd-a261-663ddab40d5e",
      "status": "OFFERED"
    }
  ]
}`

**Comment by shreeharsh-a** [comment]
@MeenaAmz

**Comment by puppsupr** [comment]
Hi [brandondev1](https://github.com/brandondev1),
Do you see then inline with what you see on the UI ?

**Comment by brandondev1** [comment]
> Hi [brandondev1](https://github.com/brandondev1), Do you see then inline with what you see on the UI ?

Not sure what you mean.  I get two different responses from the same request.  The first one above contains 3 options (all with $0 fee).  The second response contains 1 option.  The 1 option returned in the second response (placementOptionId: ple8c69e03-6ace-46bd-a261-663ddab40d5e) is one of the placement options in the first response, but the fee is $0 in the first response and $153 in the second response.

**Comment by brandondev1** [comment]
Here is one where I get 7 different placement options, but everyone of them is 2 shipments with $0 fees.  
`{
  "pagination": {},
  "placementOptions": [
    {
      "fees": [
        {
          "description": "Placement service fee represents service to inbound with minimal shipment splits and destinations of skus",
          "type": "FEE",
          "value": {
            "amount": 0,
            "code": "USD"
          },
          "target": "Placement Services"
        }
      ],
      "shipmentIds": [
        "sh8da74ec5-6653-4928-b895-56883538e576",
        "sh9afd1246-fd49-48b8-a6aa-17b65bfb5fd0"
      ],
      "discounts": [],
      "expiration": "2024-07-14T17:28Z",
      "placementOptionId": "pl041beb50-88c2-4b81-bdb2-ba2114fbf4fe",
      "status": "OFFERED"
    },
    {
      "fees": [
        {
          "description": "Placement service fee represents service to inbound with minimal shipment splits and destinations of skus",
          "type": "FEE",
          "value": {
            "amount": 0,
            "code": "USD"
          },
          "target": "Placement Services"
        }
      ],
      "shipmentIds": [
        "sh4f6ca291-a16d-4e00-a1ca-ce2fa42638fe",
        "sh650d2b71-2e2d-431d-a24f-5ba042fde343"
      ],
      "discounts": [],
      "expiration": "2024-07-14T17:28Z",
      "placementOptionId": "pl5338d962-fa8f-437d-be99-3b8aa0d2f501",
      "status": "OFFERED"
    },
    {
      "fees": [
        {
          "description": "Placement service fee represents service to inbound with minimal shipment splits and destinations of skus",
          "type": "FEE",
          "value": {
            "amount": 0,
            "code": "USD"
          },
          "target": "Placement Services"
        }
      ],
      "shipmentIds": [
        "sh5409e957-64f5-46f9-85a6-c07843a1de92",
        "sh5782849e-b620-47b5-b0a1-ac08ccbcd375"
      ],
      "discounts": [],
      "expiration": "2024-07-14T17:28Z",
      "placementOptionId": "pl5bda7f4f-fffd-4935-b425-df406505b88b",
      "status": "OFFERED"
    },
    {
      "fees": [
        {
          "description": "Placement service fee represents service to inbound with minimal shipment splits and destinations of skus",
          "type": "FEE",
          "value": {
            "amount": 0,
            "code": "USD"
          },
          "target": "Placement Services"
        }
      ],
      "shipmentIds": [
        "sh43d90a44-2455-4106-9793-63dd4d605b46",
        "sh991d7e21-3180-4ef7-9e1c-4adaccec93d9"
      ],
      "discounts": [],
      "expiration": "2024-07-14T17:28Z",
      "placementOptionId": "pl655f9210-244f-409f-85d4-d3c908b1ec61",
      "status": "OFFERED"
    },
    {
      "fees": [
        {
          "description": "Placement service fee represents service to inbound with minimal shipment splits and destinations of skus",
          "type": "FEE",
          "value": {
            "amount": 0,
            "code": "USD"
          },
          "target": "Placement Services"
        }
      ],
      "shipmentIds": [
        "sh3d5aba1c-dcdf-4caf-8a51-a0e14556f60c",
        "shd1d9b042-7ebf-4623-9e43-e33cb831220e"
      ],
      "discounts": [],
      "expiration": "2024-07-14T17:28Z",
      "placementOptionId": "plc4e6d65e-638f-401b-ad1b-66122b7b2dae",
      "status": "OFFERED"
    },
    {
      "fees": [
        {
          "description": "Placement service fee represents service to inbound with minimal shipment splits and destinations of skus",
          "type": "FEE",
          "value": {
            "amount": 0,
            "code": "USD"
          },
          "target": "Placement Services"
        }
      ],
      "shipmentIds": [
        "shaef4ad72-06fb-4508-9b9b-f4e140d27ebe",
        "she38f9e37-f72d-4ff2-9713-964b7835f650"
      ],
      "discounts": [],
      "expiration": "2024-07-14T17:28Z",
      "placementOptionId": "plc67b515e-85d4-4848-b607-345ad0324aaf",
      "status": "OFFERED"
    },
    {
      "fees": [
        {
          "description": "Placement service fee represents service to inbound with minimal shipment splits and destinations of skus",
          "type": "FEE",
          "value": {
            "amount": 0,
            "code": "USD"
          },
          "target": "Placement Services"
        }
      ],
      "shipmentIds": [
        "sh30485c43-c136-46a0-958a-48c014eb116d",
        "sh4255547f-96a8-4835-af74-bab1e4db41c1"
      ],
      "discounts": [],
      "expiration": "2024-07-14T17:28Z",
      "placementOptionId": "ple0340be0-7767-481f-95b5-fa5ad0797bcd",
      "status": "OFFERED"
    }
  ]
}`

**Comment by puppsupr** [comment]
Hi,
You can choose the option that best works for you. I was asking if you see same number of options on the seller central Send to Amazon user Interface.

**Comment by JuCongBoy** [comment]
@puppsupr Hi, I need your help sincerely.


When I first used the generatePlacementOptions interface, the data returned included scenarios for multiple shipments, as well as scenarios for a single shipment. After selecting a plan with multiple shipments, I tried to get the details of the boxes in each shipment through the listShipmentBoxes interface, but found that the box details returned were empty.

To solve this problem, I re-uploaded the packing information and called the generatePlacementOptions interface again. However, the result returned by the interface does not contain multiple packages, only the solution generated based on the latest packing information. After selecting one of the import configurations, no previous problems were encountered in subsequent operations.

Why do I see multiple shipment options after the first call to generatePlacementOptions, but after updating the packing information and re-calling the interface, I no longer see multiple shipment options? Also, why are the box details obtained through listShipmentBoxes empty after the first multi-shipment option selection?
